front/lib/iam/session.ts

```diff
@@ -270,3 +270,7 @@ export const withSuperUserAuthRequirements =
     requireUserPrivilege: "superuser",
     requireCanUseProduct: false,
   });
+
+export const withNoUserAuthRequirements = makeGetServerSidePropsRequirementsWrapper({
```

Comment:
let's use the full form and remove that helper

Reply:
Removed but not sure to understand why

Reply:
It's just more explicit. Otherwise someone will create an helper on your helper and ad the end we'll have a tower of 10 helpers before reading any useful code :)